filter_linux.go

```diff
@@ -235,6 +223,9 @@ func (h *Handle) filterModify(filter Filter, flags int) error {
 		Info:    MakeHandle(base.Priority, nl.Swap16(base.Protocol)),
 	}
 	req.AddData(msg)
+	if filter.Attrs().Chain != nil {
+		req.AddData(nl.NewRtAttr(nl.TCA_CHAIN, nl.Uint32Attr(*filter.Attrs().Chain)))
+	}
 	req.AddData(nl.NewRtAttr(nl.TCA_KIND, nl.ZeroTerminated(filter.Type())))
 
```

Comment:
in case proto is `unix.RTM_DELTFILTER` i dont think we want to add all the below RTAttrs below

Reply:
In an original implementation in iproute2 there's no specific case for deletion: [1] and [2]


[1] https://github.com/shemminger/iproute2/blob/main/tc/tc_filter.c#L761
[2] https://github.com/shemminger/iproute2/blob/main/tc/tc_filter.c#L67-L86

Reply:
ack